app/tests/integrations/google_workspace/test_google_calendar.py

```diff
@@ -154,8 +154,8 @@ def test_insert_event_no_kwargs_no_delegated_email(
         scopes=["https://www.googleapis.com/auth/calendar.events"],
         delegated_user_email="test_email",
         body={
-            "start": {"dateTime": start.isoformat(), "timeZone": "America/New_York"},
-            "end": {"dateTime": end.isoformat(), "timeZone": "America/New_York"},
+            "start": {"dateTime": start, "timeZone": "America/New_York"},
```

Comment:
So I think the timing needs to be converted to isoformat (ie something like 2020-03-20T14:28:23.382748). Is the start already in this format? If it is then it is ok, but I thought it was not. Could totally be wrong.